README.md

```diff
@@ -6,14 +6,39 @@
 ![version](https://img.shields.io/github/v/release/keptn/lifecycle-toolkit)
 [![GitHub Discussions](https://img.shields.io/github/discussions/keptn/lifecycle-toolkit)](https://github.com/keptn/lifecycle-toolkit/discussions)
 
-The Keptn Lifecycle Toolkit (KLT) provides a “cloud-native” approach
+This is the primary repository for the Keptn software and documentation.
+Keptn provides a “cloud-native” approach
 for managing the application release lifecycle
+metrics, observability, health checks,
 with pre- and post-deployment evaluations and tasks.
-It also supports application health checks,
-metrics, and observability.
 It is an incubating project, under the umbrella of the
 [Keptn Application Lifecycle working group](https://github.com/keptn/wg-app-lifecycle).
 
+> **Note**
+  The primary repository for V1 (formerly known as "Keptn") is
+  [https://github.com/keptn/keptn](https://github.com/keptn/keptn)
```

Comment:
pls remove this note for now

Reply:
I reverted to the KLT/Keptn v1 terminology for the whole README file, so this now points to the repo for Keptn v1, which is what it's called elsewhere in the docs.